packages/cuba-react-core/src/app/MainStore.ts

```diff
@@ -125,10 +125,8 @@ export class MainStore {
 
   @action
   login(login: string, password: string) {
-    return this.cubaREST.login(login, password).then(action(() => {
-      this.userName = login;
-      this.authenticated = true;
-    }))
+    return this.cubaREST.login(login, password)
+      .then(() => this.initialize());
```

Comment:
I think it's a little bit wrong since by the moment of login the app already should be initialized. Also we don't make `getUserInfo` in purpose (to prevent excessive requests), I would rather discuss returning user data (optionally) right in login response.